### DISCUSSION TITLE
Snippets?

### DISCUSSION THREAD

**Discussion by tra** [zed-industries/zed#6469] [Feature Requests]
Is there a way to implement code snippets or is it called something else like completion or is there an LSP involved?  If nothing else, I wanted to help somebody else like me who is searching for "snippets" to figure out how to do this.

**Comment by JosephTLyons** [comment]
At the moment, there is no support for snippets in Zed.  You can follow this issue to track progress on it:

- https://github.com/zed-industries/community/issues/566

**Comment by JosephTLyons** [comment]
Zed does have support for snippets now. Here is the search to loosely pull up all the PRs related to snippet support:

- https://github.com/zed-industries/zed/pulls?q=is%3Apr+snippets+is%3Aclosed+author%3Aosiewicz
